torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -477,17 +477,14 @@ void TensorPipeAgent::sendCompletedResponseMessage(
           << " is sending response to request #" << messageId << " to "
           << pipe->getRemoteName();
 
-  const c10::optional<utils::FutureError> error =
-      futureResponseMessage->error();
-  Message&& responseMessage = std::move(*futureResponseMessage).moveValue();
-  responseMessage.setId(messageId);
```

Comment:
Where did this line go? Is it not needed?

Reply:
Yep, I don't think this is necessary as request cbs have already set the id for the response message. But let me add it back for safety.